### ISSUE TITLE
Behavior similar to NpmInstall Gradle Task

### ISSUE THREAD

**Issue by nkoterba** [craigburke/bower-installer-gradle#1]
Hi -

So we wrote some gradle tasks to basically do what at initial glance your plugin looked like it might do.  So I was very happy to see this new plugin popup on Gradle's plugin repository.

However, I was expecting it to behave similar to how moowork's node/npm plugin works.

I see it creates a `bowerInstall` task, which I thought would just run `bower install` using the bower.json file in our project directory.

However, it seems that this plugin expects stuff that's normally in `bower.json` to be in our `build.gradle` file.  Furthermore, it seems like this plugin actually deletes our existing `bower.json` file upon successful execution.

Finally, when I run `gradle bowerInstall --rerun-tasks -PbowerDebug=true`, I was expecting to see the standard `bower_components` folder created in our project directory.

We reference that folder and the bower libraries in other gulp scripts so the absence of that folder breaks our other scripts.

So in summary, what we need is a gradle plugin that wraps common bower tasks: clean, list, install, etc. but uses expected bower constructs, e.g. a `bower.json` file, a default install directory of `bower_components`, etc.

Does your plugin provide this functionality?

**Comment by craigburke** [comment]
I don't think this plugin is going to meet your particular needs.

Although, here's an example of a project that just uses bower and reads the `bower.json` file:
https://github.com/jeffbrown/five15/blob/1e85812f8ca72aa82d156c08db3601e8478cb230/build.gradle

That might be closer to what you're looking for.

**Comment by nkoterba** [comment]
@craigburke Thanks for the reference! I guess for now we will continue or own implementation of "bowerInstall".

The sample `build.gradle` file you sent is great!

**Comment by craigburke** [comment]
Cool. I'm happy it helped!
